NEWS.txt

```diff
@@ -37,6 +37,10 @@ using the provided 'sstableupgrade' tool.
 4.0.2
 =====
 
+New features
+------------
+    - DCL statements in audit logs will now obscure only the password if they don't fail to parse.
```

Comment:
Is this really a new feature? I mean it sounds weird to me - also, we already were failing to obfuscate in certain parse errors

Reply:
Hi @ekaterinadimitrova2 this suggestions from adding this to NEWS.txt comes from the discussion in which you convinced me about adding it :-): `Also NEWS.txt because there will be a visible change for the users. The entries will obfuscate only the password and the DCL statements won't be that useless anymore`. See the second resolved comment from the top.

So I am +1 on it, iiuc Benjamin reviewed as so he is ok with that as well and as you don't seem violently against it I will guess it is ok to leave it. I hope I am reading this ok.